CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 CHANGES:
+* Add `server.service.activeNodePort` and `server.service.standbyNodePort` to specify the `nodePort` for active and standby services. [GH-610](https://github.com/hashicorp/vault-helm/pull/610)
```

Comment:
Seems like this is more of a feature than a change? Since these are new options?

Reply:
:+1: